installer/manifests/keptn/values.yaml

```diff
@@ -5,6 +5,12 @@ global:
     registry: docker.io/keptn
     ## @param global.keptn.tag The tag of Keptn that should be used for all images
     tag: ""
+  initContainers:
+    ## @param global.initContainers.image Init container image to enable staggered rollout of Keptn
+    image: "busybox"
```

Comment:
is there a smaller image that provides `nslookup`?

Reply:
the busybox image has 1.24MB, that's pretty hard to beat...